<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>quality_errors_dock</class>
 <widget class="QDockWidget" name="quality_errors_dock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quality errors</string>
  </property>
  <widget class="QWidget" name="quality_errors_dock_contents">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="dock_layout">
      <item>
       <layout class="QHBoxLayout" name="tree_filters_layout">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="filter_button">
          <property name="text">
           <string/>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Filters</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="primary_filters_grid" rowstretch="0,0" columnstretch="0,0,0">
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Feature type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
         <widget class="QToolButton" name="feature_type_filter">
          <property name="text">
           <string>3 active filters</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Feature attribute</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" alignment="Qt::AlignLeft|Qt::AlignVCenter">
         <widget class="QToolButton" name="feature_attribute_filter">
          <property name="text">
           <string>No active filters</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Error type</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
         <widget class="QToolButton" name="error_type_filter">
          <property name="text">
           <string>No active filters</string>
          </property>
          <property name="popupMode">
           <enum>QToolButton::InstantPopup</enum>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QgsCollapsibleGroupBox" name="mGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>15</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>100</height>
           </size>
          </property>
          <property name="title">
           <string>Extra filters (3 active)</string>
          </property>
          <widget class="QWidget" name="gridLayoutWidget">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>371</width>
             <height>61</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="secondary_filters_grid" rowstretch="0,0" columnstretch="0,0,0" rowminimumheight="0,0">
            <property name="sizeConstraint">
             <enum>QLayout::SetDefaultConstraint</enum>
            </property>
            <property name="topMargin">
             <number>10</number>
            </property>
            <item row="0" column="1">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Extra filter 2</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QToolButton" name="feature_type_filter_2">
              <property name="text">
               <string>5 active filters</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Extra filter 3</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QToolButton" name="feature_attribute_filter_2">
              <property name="text">
               <string>No active filters</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Extra filter1</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" alignment="Qt::AlignLeft|Qt::AlignVCenter">
             <widget class="QToolButton" name="error_type_filter_2">
              <property name="text">
               <string>No active filters</string>
              </property>
              <property name="popupMode">
               <enum>QToolButton::InstantPopup</enum>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>Show user processed errors</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Reset filters</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="info_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="error_tree_layout">
        <item>
         <widget class="QTreeView" name="error_tree_view"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="map_actions_layout">
        <item>
         <widget class="QCheckBox" name="filter_with_map_extent_check_box">
          <property name="text">
           <string>Show only errors within map extent</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="show_errors_on_map_check_box">
          <property name="text">
           <string>Visualize errors on map</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="close_button_layout">
        <item>
         <widget class="QPushButton" name="close_button">
          <property name="layoutDirection">
           <enum>Qt::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
